File: src/main/java/com/ject/studytrip/studylog/domain/repository/StudyLogDailyMissionRepository.java
package com.ject.studytrip.studylog.domain.repository;

import com.ject.studytrip.studylog.domain.model.StudyLogDailyMission;
import java.util.List;

public interface StudyLogDailyMissionRepository {
    List<StudyLogDailyMission> saveAll(List<StudyLogDailyMission> studyLogDailyMissions);
}

File: src/main/kotlin/com/ject/studytrip/studylog/domain/repository/StudyLogDailyMissionRepository.kt
package com.ject.studytrip.studylog.domain.repository

import com.ject.studytrip.studylog.domain.model.StudyLogDailyMission

interface StudyLogDailyMissionRepository {
    fun saveAll(studyLogDailyMissions: List<StudyLogDailyMission>): List<StudyLogDailyMission>
}
